modules/deploy/pages/deployment-option/cloud/managed-connectors/create-s3-sink-connector.adoc

```diff
@@ -143,7 +143,7 @@ a| The template for file names on S3. Supports `{{ variable }}` placeholders for
 
 | `Envelope for primitives`
 | `format.output.envelope`
-| Whether to enable envelope for entries with single field.
+| Whether to enable additional JSON object wrapping the actual value.
```

Comment:
Specifies whether or not to enable additional JSON object wrapping of the actual value.